features/archive_autocomplete.feature

```diff
@@ -34,3 +34,14 @@ Feature: Archive autocomplete
             {"value": "PUBLISHED-C"}
         ]}
         """
+        When we get "/archive_autocomplete?field=slugline&language=en&resources=archive"
+        Then we get list with 3 items
+        """
+        {"_items": [
+            {"value": "PUBLISHED-A"},
+            {"value": "PUBLISHED-B"},
+            {"value": "PUBLISHED-C"}
```

Comment:
might be useful to add the document count to the payload so we could sort it also based on that